### PR TITLE
[T4A6][W13-B4] Conan Kian

### PR DESCRIPTION
**Pull request by ckjr**
Ready for review

### CODE REVIEW

File: src/seedu/addressbook/common/UtilsTest.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+    @Test
+    public void nullObject_returnsTrue() {
+        final Object[] inputs = { "", null, "\n  \n" };
+        final boolean resultMessage = true;
+        assertEquals(resultMessage, Utils.isAnyNull(inputs));
+    }
+    
+    @Test
+    public void noNullObject_returnsFalse() {
```
Comment (okkhoy) [inline src/seedu/addressbook/common/UtilsTest.java:24]: the test name should have 3 parts `<method being tested>_<what is being tested>_<what is the expected return value>`
you can eliminate part 3 and 2 only if you have broadened the scope of the test to include testing multiple things.

File: src/seedu/addressbook/common/UtilsTest.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    @Test
+    public void checkUnique_returnsTrue() {
+        final List<String> inputs = Arrays.asList("A", "B", "C", "D");
+        final boolean resultMessage = true;
+        assertEquals(resultMessage, Utils.elementsAreUnique(inputs));
+    }
+    
+    @Test
+    public void checkUnique_returnsFalse() {
+        final List<String> inputs = Arrays.asList("A", "B", "C", "D", "A");
```
Comment (okkhoy) [inline src/seedu/addressbook/common/UtilsTest.java:39]: going further you can consider moving all the steps necessary for the test to run (or preconditions) to a set up method before beginning the tests; correspondingly can put all the tear down in a method after the tests.

**Comment by okkhoy** [comment]
@ckjr 
Some comments added. Please ack & close the PR after reading comments.

**Comment by ckjr** [comment]
Acknowledged.

**Pull request closed without merging**